include/cetl/pf20/cetlpf.hpp

```diff
@@ -0,0 +1,60 @@
+/// @file
+/// CETL polyfill header for C++20 types
+///
+/// @copyright
+/// Copyright (C) OpenCyphal Development Team  <opencyphal.org>
+/// Copyright Amazon.com Inc. or its affiliates.
+/// SPDX-License-Identifier: MIT
+///
+/// @warning
+/// This header violates AUTOSAR-14 M7-3-6, A17-0-1, and A16-0-1 (and possibly other rules). Don't use CETL polyfill
+/// headers in AUTOSAR code.
+///
+/// CETL polyfill headers will provide the CETL type as an aliased standard type in the std namespace when compiling
+/// using older C++ libraries and will automatically provide types from the standard library when compiling
+/// using newer C++ libraries. This allows for causal use of CETL as a true polyfill library but does violate certain
+/// coding standards. As such, for more critical software we recommend not using these headers but including the
+/// types you use from `cetl::pf20` directly.
+///
+/// For example, using the polyfill header an program like this can be written:
+///
+/// @include example_01_polyfill.cpp
+///
+/// In that example the code automatically switches to using the C++ 20 version of span when compiling
+/// with that standard enabled. For more critical code you will need to do something like this:
+///
+/// @include example_02_polyfill.cpp
+///
+/// In that example the span type can be easily redefined but it will not be automatically redefined.
+/// In these cases something like this might also be in order:
+///
+/// ```
+/// static_assert(__cplusplus < CetlCppStandard20, "Don't use CETL if you are compiling for C++20.");
+/// ```
+///
+
+#ifndef CETL_PF20_H_INCLUDED
+#define CETL_PF20_H_INCLUDED
+
+#include "cetl/cetl.hpp"
+
+#if (__cplusplus >= CetlCppStandard20)
+#    include <span>
```

Comment:
Is this std:: style?

Reply:
Yeah. @pavel-kirienko , I'm getting the right clang-format style here right?

Reply:
Seems so.